Doc/library/tempfile.rst

```diff
@@ -183,6 +183,10 @@ The module defines the following user-callable items:
       *suffix* and *prefix* now accept and default to ``None`` to cause
       an appropriate default value to be used.
 
+   .. versionchanged:: 3.6
+      The *dir* parameter now accepts objects implementing the
```

Comment:
Perhaps this and below could be simplified with :term:`path-like object`? I haven't checked the markup an example https://docs.python.org/3.8/library/os.html#os.chdir . I see this version on newer docs and hence my personal preference.

The *dir* parameter now accepts :term:`path-like object`

Reply:
done, thanks!